music_album.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
+    super(publish_date, archived: false)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    parent_result = super
+    parent_result && on_spotify
+  end
+
+  def add_music_album; end
```

Comment:
Kindly remove this method since it is not part of the requirement.

Reply:
resolved